.github/workflows/gaia.yml

```diff
@@ -1,70 +0,0 @@
-name: GraphScope GAIA CI
```

Comment:
If you intend to disable this CI temporarily, it would be better to comment out rather than remove this file.

Reply:
The GAIA ci has already been disabled before (see line 23) and not used anymore, as the related codepath has already been dropped.

Reply:
okay, I have recovered the gaia.yaml